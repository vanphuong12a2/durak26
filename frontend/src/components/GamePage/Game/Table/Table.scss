.table {
  position: relative;

  &:before {
    content: "";
    display: block;
    padding-top: 60%;
  }

  .table-inside {
    position: absolute;
    top: 0;
    left: 0;
    bottom: 0;
    right: 0;
    padding: 5px;

    .table-content {
      background-color: #26a69a;
      border-radius: 15px;
      height: 100%;
    }
  }
}
