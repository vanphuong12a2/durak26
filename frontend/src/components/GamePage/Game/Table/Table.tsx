import React from 'react';

const Table = () => {
  return (
    <div className='table'>
      <div className='table-inside'>
        Table
      </div>
    </div>
  );
};

export default Table;
