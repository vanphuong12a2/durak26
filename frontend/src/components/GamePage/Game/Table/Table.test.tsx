import React from 'react';
import {create} from 'react-test-renderer';
import Table from './Table';

describe('<Table />', () => {

  describe('Snapshots', () => {
    it('renders correctly', () => {
      const app = create(<Table/>);
      expect(app.toJSON()).toMatchSnapshot();
    });
  });

});
