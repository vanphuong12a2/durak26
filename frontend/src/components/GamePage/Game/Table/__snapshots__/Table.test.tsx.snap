// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table /> Snapshots renders correctly 1`] = `
<div
  className="table"
>
  <div
    className="table-inside"
  >
    Table
  </div>
</div>
`;
